pkg/handlers/system_intake_action_test.go

```diff
@@ -46,17 +50,38 @@ func (s HandlerTestSuite) TestSystemIntakeActionHandler() {
 		s.Equal(http.StatusCreated, rr.Code)
 	})
 
+	s.Run("POST fails with empty request body", func() {
+		rr := httptest.NewRecorder()
+		req, reqErr := http.NewRequestWithContext(
+			requestContext,
+			"POST",
+			fmt.Sprintf("/system_intake/%s/action/SUBMIT", id.String()),
```

Comment:
Should this be:

```suggestion
			fmt.Sprintf("/system_intake/%s/action", id.String()),
```

since we aren't passing the action type in the URL anymore?